mmdet/models/dense_heads/rpn_head.py

```diff
@@ -134,20 +136,26 @@ def _get_bboxes_single(self,
                 scores = rpn_cls_score.softmax(dim=1)[:, 0]
             rpn_bbox_pred = rpn_bbox_pred.permute(1, 2, 0).reshape(-1, 4)
             anchors = mlvl_anchors[idx]
-            if cfg.nms_pre > 0 and scores.shape[0] > cfg.nms_pre:
+            if cfg.nms_pre > 0:
```

Comment:
Will this change affect model's GPU inference?

Reply:
model's gpu inference will go through the elif branch, and it will not affect the original logic

Reply:
> Will this change affect model's GPU inference?

model's gpu inference will go through the elif branch, and it will not affect the original logic